lib/node_modules/@stdlib/_tools/changelog/generate/README.md

```diff
@@ -66,12 +76,50 @@ The following release types are supported:
 -   `auto`: automatically determine the release type based on parsed commit messages.
 -   `none`: no release (equivalent to not specifying a release type).
 
+By default, the function generates a `grouped` changelog for namespace packages and an `aggregated` changelog for non-namespace packages. To specify a desired output format, set the `format` option:
+
+```javascript
+// Changelog grouped by individual packages:
+var changelog = generate( '@stdlib/math/base/utils', {
+    'format': 'grouped'
+});
+// returns {...}
+
+// Changelog where all commits, potentially touching many different packages, are merged together:
+changelog = generate( '@stdlib/math/base/utils', {
+    'format': 'aggregated'
+});
+// returns {...}
+```
+
+When generating a changelog, the function uses `git log` to retrieve the commits from which to assemble a set of changes. The `flags` option allows passing options to directly to the `git log` command (e.g., to generate a changelog for a specified time interval or for an individual contributor).
+
+<!-- eslint-disable stdlib/doctest -->
+
+```javascript
+var changelog = generate( '@stdlib/ndarray', {
+    'flags': {
+        'since': 'last month'
+    }
+});
+// returns {...}
+
+changelog = generate( '@stdlib/ndarray', {
+    'flags': {
+        'author': 'Athan Reines <kgryte@gmail.com>'
+    }
+});
+// returns {...}
+```
+
 </section>
 
 <!-- /.usage -->
 
 <section class="notes">
 
+-   When `flags` are supplied, the generated changelog does not include empty releases and uses a flat output format.
```

Comment:
Is this note still accurate? It is not clear to me from the diff of `main.js` that this is true or that this constraint is necessary. I am mainly referring to the "flat" output format.

Reply:
Also, it is not clear what the "flat" output format actually refers to.

Reply:
Good catch! Removed the note.